site/biome.json

```diff
@@ -1,6 +1,6 @@
 {
 	"files": {
-		"ignore": ["**/*Generated.ts"]
+		"ignore": ["**/*Generated.ts", "../helm/**/templates/*.yaml"]
```

Comment:
this shouldn't be necessary, we only run biome from within site/

Reply:
I am running it on the `*.yaml` in `helm` too and want to ignore the templates folder as that is not a valid YAML.